vexim/adminuserchangesubmit.php

```diff
@@ -146,7 +146,7 @@
       header ("Location: adminuser.php?failupdated={$_POST['localpart']}");
       die;
     }
-  } else if ($_POST['clear'] != $_POST['vclear']) {
+  } else if (!alias_validate_password($_POST['clear'], $_POST['vclear'])) {
```

Comment:
A call to an `alias_something` function here looks a bit dumb, don't you think? I think it would be better to just enclose this `if..else` block in a parent `if`, similar to what's being done in `adminaliaschangesubmit.php`.